README.md

```diff
@@ -420,31 +279,215 @@ The temperature data is stored in files under data folder. One file is made for
     tail -f ./<name-of-the-lates-data-file>.txt
     ```
 
-## Usage
+## Set up the iot-lab-nodes
+
+### IoT-lab account
 
-To use this project, follow these guidelines:
+IoT-lab account is requiret to work with iot-lab. It can be requested by filling a sign up form. More information about iot-lab account can be found from iot-lab documentation.
+
+* [sign-up form](https://www.iot-lab.info/testbed/signup)
+* [more information about iot-lab accounts](https://iot-lab.github.io/docs/getting-started/user-account/)
+
+### SSH connection to iot-lab
+
+SSH connection to the iot-lab sites can be created by following iot-lab documentation about it:
+
+* [IoT-lab SSH documentation](https://iot-lab.github.io/docs/getting-started/ssh-access/)
+
+### Build the project
+
+In this tutorial we use grenoble iot-lab site. Other sites with IoT-LAB M3 board could be also used. More information about iot-lab sites can be cound from iot-lab documentation.
+
+* [iot-lab sites](https://iot-lab.github.io/docs/deployment/grenoble/)
+
+1. Connect to grenoble.iot-lab.info:
+
+    ```batch
+    ssh <username>@grenoble.iot-lab.info
+    ```
+
+2. Clone the repository to your folder of choise at grenoble.iot-lab.info:
+
+    ```bash
+    git clone https://github.com/matluuk/IoT-miniproject-1.git
+    ```
+
+3. Init the RIOT-submodule:
+
+    ```bash
+    git submodule init
+    ```
+
+4. Update the RIOT-submodule:
+
+    ```bash
+    git submodule update
+    ```
+
+5. Change into the miniproject-1 repository working directory:
+
+    ```bash
+    cd IoT-miniproject-1/app
+    ```
+
+6. Set source:
+
+    ```bash
+    source /opt/riot.source
+    ```
+
+7. Build the project:
+
+    ```bash
+    make
+    ```
+
+
+### Set up Fit IoT-Lab experiment
+
+Set up Fit IoT-Lab experiment with two nodes for device firmware and border router:
 
 1. Connect to grenoble.iot-lab.info IF not already:
 
     ```batch
     ssh <username>@grenoble.iot-lab.info
     ```
 
-2. Change into the miniproject-1 repository working directory:
+2. Submit an experiment for two nodes at grenoble for 20 minutes:
```

Comment:
experiment time should be longer for example 60 min

Reply:
Fixed